virtcontainers/experimental/README.md

```diff
@@ -1,5 +1,10 @@
 # Experimental package description
 
+* [1. What are "experimental" features?](#1-what-are-experimental-features)
```

Comment:
Heh, bullets in titles - not sure we do that anywhere else, and I can just see an update nightmare coming down the road. Would you consider dropping those whilst you are here?

Reply:
Agreed - done.